### ISSUE TITLE
Error Running InstallFullDB.sh For Classic Database

### ISSUE THREAD

**Issue by jimmybrancaccio** [cmangos/issues#1993]
## 🐛 Bugreport

When running `InstallFullDB.sh` there's an error about altering a routine.

```
$ ./InstallFullDB.sh 

Welcome to the Classic-DB helper InstallFullDB.sh

ATTENTION: Your database classicmangos will be reset to Classic-DB!
Please bring your repositories up-to-date!
Press CTRL+C to exit
...........
> Processing Classic database v1.10.0 'Echoes of War'.
ERROR 1370 (42000) at line 993960: alter routine command denied to user 'mangos'@'localhost' for routine 'classicmangos.script_npc'
ERROR: cannot apply Full_DB/ClassicDB_1_10_0_z2732.sql
```

You may also note some output [I put in Discord](https://discordapp.com/channels/362206349339262976/362206349339262978/592790920823439360).

### Expected behavior

I would expect running this script would update the database without any issues or errors.

### Version & Environment

Client Version: "1.12.1" (CLASSIC)

CMaNGOS Repo & Commit Hash:  d7e5dc55b43444c1759ad28eadff0f22b1caba6d

Database Repo & Commit Hash: b2159b6e050c23333e7ceb10d8a5746cbb4d6177

Operating System: Ubuntu Linux 18.04

### Steps to reproduce

1. Fetch given commits of classic and the classic database.
2. Make sure you've run the base SQL Stuff to get the DB structure going.
3. Setup and run the InstallFullDB.sh script.
4. Observe the error about the routine.
5. 💰 

### Crashlog

- None

```
$ mysql -V
mysql  Ver 14.14 Distrib 5.7.26, for Linux (x86_64) using  EditLine wrapper
```

**Comment by berserkingyadis** [comment]
Pls edit in the database you are using. MSQL5/8? MariaDB?

**Comment by berserkingyadis** [comment]
Can't reproduce. The DB update goes through with no problems on these commits.

My machine is Windows 10 64bit with MYSQL Ver 15.1 Distrib 10.3.11-MariaDB, for Win64 (AMD64), source revision fe0e65dbe28f93bfc677d12be4166415dc1bec5a

**Comment by jimmybrancaccio** [comment]
```
$ mysql -V
mysql  Ver 14.14 Distrib 5.7.26, for Linux (x86_64) using  EditLine wrapper
```

**Comment by jimmybrancaccio** [comment]
Just retested using:

core version - faeaefcfca28f21a6eece68a08b731de06f0aed1
db version - b2159b6e050c23333e7ceb10d8a5746cbb4d6177

and it fails:

```
$ ./InstallFullDB.sh 

Welcome to the Classic-DB helper InstallFullDB.sh

ATTENTION: Your database classicmangos will be reset to Classic-DB!
Please bring your repositories up-to-date!
Press CTRL+C to exit
...........
> Processing Classic database v1.10.0 'Echoes of War'.
ERROR 1370 (42000) at line 993960: alter routine command denied to user 'mangos'@'localhost' for routine 'classicmangos.script_npc'
ERROR: cannot apply Full_DB/ClassicDB_1_10_0_z2732.sql
```

**Comment by jimmybrancaccio** [comment]
After some further investigation and testing we've determined this was an issue on my end. This was related to a Discord discussion:

https://discordapp.com/channels/362206349339262976/362206349339262978/565743824781967371

I had updated my `sql/create/db_create_mysql.sql` and adjust the GRANT query for the `classicmangos`database to remove the `ROUTINE` stuff so the update couldn't run because of missing permissions. I ran the original GRANT query with the ROUTINE permission and then running `InstallFullDB.sh` worked perfectly fine.

We were wondering why ROUTINES are used in classic only but couldn't come up with anything. Maybe @cala knows?

Moral of the story, don't change the GRANTS if you don't know what you're doing! 😝

**Comment by evil-at-wow** [comment]
It looks like cyberium and Tobschinski have fixed this in cmangos/classic-db@c705de2f.

And to answers your question: I checked the ClassicDB v1.10 release I have here, and that fragment that Tobschinski updated is **not** there in my version (the official release), so classic is also **not** using routines. It looks like that part was introduced when @cala re-exported it in cmangos/classic-db@9e586c2029, so it's probably a routine that he has locally in his database and that somehow made it in the re-export.

**Comment by jimmybrancaccio** [comment]
Ah cool, thanks for the followup @evil-at-wow, appreciate it!
